content/en/docs/concepts/services-networking/topology-aware-hints.md

```diff
@@ -0,0 +1,127 @@
+---
+reviewers:
+- robscott
+title: Topology Aware Hints
+feature:
+  title: Topology Aware Hints
+  description: >
+    Enables topology aware routing with topology hints on EndpointSlices.
```

Comment:
This is too specific to be a front-page website feature, I'm afraid. Please _remove_ that feature card.

Reply:
Woops, didn't realize what this did. Was just copied from another page, thanks for catching this!